src/DynamoCoreWpf/Views/Preview/Watch3DView.xaml.cs

```diff
@@ -192,14 +168,40 @@ public double LightElevationDegrees
             set { lightElevationDegrees = value; }
         }
 
-#if DEBUG
-        /// <summary>
-        /// The TestSelectionCommand is used in the WatchSettingsControl
-        /// to test the ability to toggle a boolean effect variable
-        /// representing the selection state.
-        /// </summary>
-        public Dynamo.UI.Commands.DelegateCommand TestSelectionCommand { get; set; }
-#endif
+        private Dictionary<string, Model3D> geometryDictionary= new Dictionary<string, Model3D>();
+
+        public Dictionary<string, Model3D> GeometryDictionary
```

Comment:
@ramramps Can we rename this to `Model3DDictionary`? It contains more than just geometry.